src/main/java/de/slimecloud/slimeball/features/youtube/YoutubeListener.java

```diff
@@ -0,0 +1,31 @@
+package de.slimecloud.slimeball.features.youtube;
+
+import de.cyklon.jevent.EventHandler;
+import de.cyklon.jevent.Listener;
+import de.slimecloud.slimeball.main.SlimeBot;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import net.dv8tion.jda.api.entities.channel.concrete.TextChannel;
+
+@Slf4j
+@Listener
+@RequiredArgsConstructor
+public class YoutubeListener {
+	private final SlimeBot bot;
+
+	@EventHandler
+	public void onUpload(YoutubeVideoEvent event) {
+		bot.getJda().getGuilds().forEach(g -> bot.loadGuild(g).getYoutube().ifPresent(c -> {
+			TextChannel channel = bot.getJda().getTextChannelById(c.getChannel());
+
+			if (channel != null) {
+				String msg = event.isLive() ? c.getLiveMessage() : c.getVideoMessage();
+
+				channel.sendMessage(msg
+						.replace("%role%", String.format("<@&%s>", c.getRole()))
```

Comment:
Use Role#getAsMention

Reply:
I deliberately did it this way to save an unnecessary request to the discord api.

Reply:
That would not require an request because roles are cached